packages/marketplace/src/reducers/revision-detail.ts

```diff
@@ -57,6 +57,8 @@ const revisionDetailReducer = (state: RevisionDetailState = defaultState, action
       loading: false,
       error: false,
       revisionDetailData: action.data,
+      approveFormState: 'PENDING',
```

Comment:
Does it same `'PENDING'` like `declineFormState`

Reply:
Yes, they're using the same values. This is the defaultState:
```
export const defaultState: RevisionDetailState = {
  loading: false,
  error: false,
  revisionDetailData: null,
  approveFormState: 'PENDING',
  declineFormState: 'PENDING',
}
```